frontend/components/AccessibleAlignmentSelector.cpp

```diff
@@ -0,0 +1,179 @@
+/******************************************************************************
+    Copyright (C) 2025 by Taylor Giampaolo <warchamp7@obsproject.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include "AccessibleAlignmentSelector.hpp"
+#include <OBSApp.hpp>
+
+namespace {
+static const char *indexToString[] = {
+	"Basic.TransformWindow.Alignment.TopLeft",    "Basic.TransformWindow.Alignment.TopCenter",
+	"Basic.TransformWindow.Alignment.TopRight",   "Basic.TransformWindow.Alignment.CenterLeft",
+	"Basic.TransformWindow.Alignment.Center",     "Basic.TransformWindow.Alignment.CenterRight",
+	"Basic.TransformWindow.Alignment.BottomLeft", "Basic.TransformWindow.Alignment.BottomCenter",
+	"Basic.TransformWindow.Alignment.BottomRight"};
+}
+
+AccessibleAlignmentSelector::AccessibleAlignmentSelector(AlignmentSelector *widget)
+	: QAccessibleWidget(widget, QAccessible::Grouping),
+	  widget(widget)
+{
+	for (int i = 0; i < 9; ++i) {
+		AccessibleAlignmentCell *cell = new AccessibleAlignmentCell(this, widget, i);
+		QAccessible::registerAccessibleInterface(cell);
+		cellInterfaces.insert(i, QAccessible::uniqueId(cell));
+	}
+}
+
+AccessibleAlignmentSelector::~AccessibleAlignmentSelector()
+{
+	for (QAccessible::Id id : std::as_const(cellInterfaces)) {
+		QAccessible::deleteAccessibleInterface(id);
+	}
+}
+
+int AccessibleAlignmentSelector::childCount() const
+{
+	return 9;
```

Comment:
Why is this hard-coded to `9`?

Reply:
Yep magic numbers should be avoided. If this is indeed a constant value, a `constexpr` should be used with a descriptive name, which will also logically couple the usage of the constant in other places in the code.